knip.ts

```diff
@@ -11,6 +11,7 @@ export default {
     types: 'off',
     unresolved: 'off',
   },
+  vitest: { config: ['vitest.config.mts', 'packages/**/vitest.config.mts'] },
```

Comment:
We just have the one level of packages:

```suggestion
  vitest: { config: ['vitest.config.mts', 'packages/*/vitest.config.mts'] },
```

Reply:
Done!